sc/social/like/subscribers.py

```diff
@@ -107,3 +113,40 @@ def assign_canonical_url(obj, event):
             'Canonical domain not set in Social Media configlet; '
             "Facebook's Open Graph canonical URL (og:orl) will not be available"
         )
+
+
+def social_content_check(obj, event):
+    """Check if content is valid if enabled in sc.social.content, in action when edit."""
+    request = getRequest()
+
+    # FIXME: view still available with package uninstalled; why?
```

Comment:
The error occurs when you install a Plone site.
I also tested `zcml:condition=" installed sc.social.like`, but it did not work.
I also tested `zcml:condition="have sc.social.like`, but it did not work.